docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -224,6 +244,15 @@ public DockerImagePullStrategy getPullStrategy() {
     public void setPullStrategy(DockerImagePullStrategy pullStrategy) {
         this.pullStrategy = pullStrategy;
     }
+    
+    public List<? extends NodeProperty<?>> getNodeProperties() {
```

Comment:
This return type is meant to match the constructor parameter type of AbstractCloudSlave. Removing the usage of the generic wildcard type here would require a cast at the super() call in DockerSlave. 

See [AbstractCloudSlave](http://javadoc.jenkins-ci.org/hudson/slaves/AbstractCloudSlave.html#AbstractCloudSlave(java.lang.String, java.lang.String, java.lang.String, int, hudson.model.Node.Mode, java.lang.String, hudson.slaves.ComputerLauncher, hudson.slaves.RetentionStrategy, java.util.List)): 
`public AbstractCloudSlave(String name,
                  String nodeDescription,
                  String remoteFS,
                  int numExecutors,
                  Node.Mode mode,
                  String labelString,
                  ComputerLauncher launcher,
                  RetentionStrategy retentionStrategy,
                  List<? extends NodeProperty<?>> nodeProperties)
                   throws Descriptor.FormException,
                          IOException
`